Extensions/TweenBehavior/JsExtension.js

```diff
@@ -44,10 +44,10 @@ module.exports = {
 
     const behavior = extension
       .addBehavior(
-        "TweenBehavior",
-        _("Tween animation"),
-        "TweenBehavior",
-        _("Tween animation"),
+        "Tween",
```

Comment:
This should actually be TweenBehavior ;) It's the internal name stored in the project file and used to identify the behavior in the engine, so better keep TweenBehavior

Reply:
Ups saw this too late. Will revert the changes I just committed.